### PR TITLE
add word cloud

### PR DESCRIPTION
**Pull request by netashefer**

### CODE REVIEW

File: src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx [hunk 1]
```diff
@@ -17,7 +18,7 @@ import './ParametersPanel.scss';
 const ParametersPanel = () => {
 	const dataSources = useRecoilValue(dataSourcesAtom);
 	const [dataSource, setDataSource] = useState<OptionItem>(null);
-	const [graphType, setGraphType] = useState<OptionItem>(null); //for the time being all of these are useState
+	const [graphType, setGraphType] = useState<OptionItem<GraphType>>(null); //for the time being all of these are useState
```
Comment (YahelR) [inline src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx]: I don't know which one of us will merge before the other one, just make sure the word cloud works fine both in creation mode and edit mode

File: src/types/utility.types.ts [hunk 2]
```diff
@@ -1 +1,11 @@
-export type setState<T> = React.Dispatch<React.SetStateAction<T>>;
\ No newline at end of file
+export type setState<T> = React.Dispatch<React.SetStateAction<T>>;
+export type Subset<K> = { // nested Partial - https://grrr.tech/posts/2021/typescript-partial/
```
Comment (YahelR) [inline src/types/utility.types.ts:2]: wowie

**Comment by YahelR** [comment]
approved

File: src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx (hunk 1 above)
Comment (netashefer) [reply src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx]: I'll wait for your merge first and I make it editable

**Pull request merged**